### PR TITLE
Change wait to sleep in buildahimage readme

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Change wait to sleep 100000 as we've had some reports of problems
with wait.  Fixes: #1665

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

File: contrib/buildahimage/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ mkdir /var/lib/mycontainer
 # Run the image detached using the host's network in a container name
 # buildahctr, turn off label and seccomp confinement in the container
 # and then do a little shell hackery to keep the container up and running.
-podman run --detach --name=buildahctr --net=host --security-opt label=disable --security-opt seccomp=unconfined --device /dev/fuse:rw -v /var/lib/mycontainer:/var/lib/containers:Z  stable sh -c 'while true ;do wait; done'
+podman run --detach --name=buildahctr --net=host --security-opt label=disable --security-opt seccomp=unconfined --device /dev/fuse:rw -v /var/lib/mycontainer:/var/lib/containers:Z  stable sh -c 'while true ;do sleep 100000 ; done'
```
Comment (vrothberg) [inline contrib/buildahimage/README.md]: Can we do a `sleep infinity` instead?

Comment (TomSweeneyRedHat) [reply contrib/buildahimage/README.md]: I changed it to a limited time as the reporter had said things were going wonky on his system if he forgot to kill the container after testing.

Comment (vrothberg) [reply contrib/buildahimage/README.md]: Ah, make sense, thanks!

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**